k8s/kustomize/helm-components-sc/patch_controller.yaml

```diff
@@ -5,8 +5,7 @@ metadata:
 spec:
   template:
     spec:
-      imagePullSecrets:
-      - name: '{{ .Values.imagePullSecret }}'
+      imagePullSecrets: []
```

Comment:
why it is not parameterised anymore?

Reply:
to get ability for full list is not possible (as far as I can tell) using Kustomize so I added to `k8s/Makefile` `sed -i 's/\(.*\)\(imagePullSecrets:\).*/{{- with .Values.imagePullSecrets }}\n\1\2\n\1{{- toYaml . | nindent 8 }}\n{{- end }}/g' ${HELM_COMPONENTS_BASE}/seldon-v2-components.yaml`